ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -105,6 +105,26 @@ EOF
 
 echo "CatalogSource name is \"$CATSRC\""
 
+IS_CATSRC_CREATED=false
+# Wait for 10 minutes until the Catalog source state is 'READY'
+for i in $(seq 1 120); do
+    CATSRC_STATE=$(oc get catalogsources/"$CATSRC" -n "$OO_INSTALL_NAMESPACE" -o jsonpath='{.status.connectionState.lastObservedState}')
+    echo $CATSRC_STATE
+    if [ "$CATSRC_STATE" = "READY" ] ; then
+	echo "CATALOG SOURCE CREATED SUCCESSFULLY AFTER WAITING FOR $((5*i)) SECONDS"
```

Comment:
Can you change the text formatting to be in line with the rest of the script?
The other messages do not use the all uppercase format.